plugin-locationcomponent/src/main/java/com/mapbox/maps/plugin/locationcomponent/LocationComponentPluginImpl.kt

```diff
@@ -84,6 +88,36 @@ class LocationComponentPluginImpl : LocationComponentPlugin, LocationConsumer,
     onIndicatorBearingChangedListener.remove(listener)
   }
 
+  /**
+   * Check is a given point is on the rendered location puck.
+   *
+   * @param point the point to validate
+   * @param listener Listener that gets invoked when the validation finished.
+   */
+  override fun isPointOnLocationPuck(point: Point, listener: IsPointOnLocationPuckListener) {
```

Comment:
Capturing from chat that you were looking for some alternative names:

    isAtPoint
    isOnPoint
    isLocatedAt
    matchesPoint
    geometryEquals
    pointEquals
    isLocatedAt
    isPointAt
    ...
    :)

Reply:
I vote for `isLocatedAt`.

Reply:
👍